system/Cache/Handlers/WincacheHandler.php

```diff
@@ -179,7 +179,7 @@ public function decrement(string $key, int $offset = 1)
 	 *
 	 * @return boolean
 	 *
-	 * @codeCoverageIgnore
+	 * @boolean Success or failure
```

Comment:
```suggestion
	 * @return boolean Success or failure
```

Reply:
I pasted over the wrong docblock - return statement above. 😳 If you can fix this in a suggest I can merge it here (or you can) but I won't be on desktop for a while.